testing/downloadLivy.sh

```diff
@@ -17,12 +17,20 @@
 #
 
 if [[ "$#" -ne 1 ]]; then
-    echo "usage) $0 [livy version]"
+    echo "usage) $0 <livy version> [scala version]"
     echo "   eg) $0 0.2"
+    echo "       $0 0.8 2.11"
     exit 0
 fi
 
+# See simple version normalization:
+# http://stackoverflow.com/questions/16989598/bash-comparing-version-numbers
+function version { echo "$@" | awk -F. '{ printf("%03d%03d%03d\n", $1,$2,$3); }'; }
 LIVY_VERSION="${1}"
+SCALA_VERSION_SUFFIX=""
+if [ $(version $LIVY_VERSION) -ge $(version "0.8.0") ]; then
+    SCALA_VERSION_SUFFIX="_${2}"
```

Comment:
since 0.8.0, livy binary zip contains scala suffix
```
https://dist.apache.org/repos/dist/release/incubator/livy/0.7.1-incubating/apache-livy-0.7.1-incubating-bin.zip
https://dist.apache.org/repos/dist/release/incubator/livy/0.8.0-incubating/apache-livy-0.8.0-incubating_2.11-bin.zip
```